src/applications/hca/config/form.js

```diff
@@ -186,10 +188,15 @@ const attachmentsSchema = {
 // For which page needs prefill-message, check
 // vets-api/config/form_profile_mappings/1010ez.yml
 const formConfig = {
+  rootUrl: manifest.rootUrl,
   urlPrefix: '/',
   submitUrl: `${environment.API_URL}/v0/health_care_applications`,
   trackingPrefix: 'hca-',
   formId: VA_FORM_IDS.FORM_10_10EZ,
+  benefitDescription: {
+    benefitType: 'health care benefits',
+    subText: '10-10EZ',
```

Comment:
Unfortunately, because the `formId` is used as a key in saved form lookup, it cannot be changed after the form is in production without synchronizing the migration of all its saved forms to use the new `formId`. That's outside the scope of this PR.